### PR TITLE
MAINT xfail remaining Chrome 89 fatal errors

### PR DESCRIPTION
**Pull request by hoodmane**
See #1493 and #1495.

### CODE REVIEW

File: packages/pandas/test_pandas.py [hunk 1]
```diff
@@ -51,6 +51,10 @@ def test_extra_import(selenium, request):
 @pytest.mark.skip_refcount_check
 def test_load_largish_file(selenium_standalone, request, httpserver):
     selenium = selenium_standalone
+    if selenium.browser == "chrome":
+        pytest.xfail(
+            "test_load_largish_file triggers a fatal runtime error in Chrome 89 see #1495"
```
Comment (rth) [inline packages/pandas/test_pandas.py:56]: FWIW, this also used to fail a long time ago in Chrome.

**Review by rth — APPROVED** [review]
Thanks!

File: packages/pandas/test_pandas.py (hunk 1 above)
Comment (hoodmane) [reply packages/pandas/test_pandas.py:56]: Really weird stuff, see #1495.

**Comment by hoodmane** [comment]
`test_numcodecs` also is a persistent failure due to recursion errors, I think if I xfail that too then packages-chrome can pass. It makes sense that chrome would have more recursion errors since it's stack is 2.5x smaller than the firefox stack.

**Pull request merged**